src/main/java/com/denizenscript/ddiscordbot/events/DiscordNewUserScriptEvent.java

```diff
@@ -0,0 +1,88 @@
+package com.denizenscript.ddiscordbot.events;
+
+import net.aufdemrand.denizencore.events.ScriptEvent;
+import net.aufdemrand.denizencore.objects.Element;
+import net.aufdemrand.denizencore.objects.dObject;
+import net.aufdemrand.denizencore.scripts.containers.ScriptContainer;
+import net.aufdemrand.denizencore.utilities.CoreUtilities;
+import sx.blah.discord.handle.impl.events.guild.member.UserJoinEvent;
+
+public class DiscordNewUserScriptEvent extends ScriptEvent {
+    public static DiscordNewUserScriptEvent instance;
+
+    // <--[event]
+    // @Events
+    // discord user join (for <bot>)

```

Comment:
`joins` not `join`

Reply:
Do you also want line 34 have the same change as above
![image](https://user-images.githubusercontent.com/8504075/53016862-eaa0a200-344e-11e9-83d5-73d1b19c4340.png)
so its like this:
`return CoreUtilities.toLowerCase(s).startsWith("discord user joins");`

Reply:
All usages of the name, yes